.github/workflows/update-gh-pages.yml

```diff
@@ -1,17 +1,40 @@
-
-name: Update index.html in GH Pages
+name: Custom Github pages post-processing
 
 on:
   workflow_dispatch:
-  workflow_call:
```

Comment:
Hmm... I didn't know this was already here. However, I am not sure if we are triggering this workflow from another one. Maybe @RobPasMue can clarify about this.

Reply:
@jorgepiloto maybe we should just leave it then? No harm in doing that.

Reply:
We are calling it from the main workflow as well if I remember correctly - please revert